composer.json

```diff
@@ -22,12 +22,12 @@
         }
     },
     "require": {
-        "php": ">=5.0",
+        "php": ">=5.5",
         "ext-shmop": "*"
```

Comment:
This is not good. This change break BC and require major release.
I do not want to throw old versions of PHP specifically in this project.
In this project, i want to keep support for legacy versions of PHP as long as possible.

Reply:
Agree. Revert this.

Reply:
To make sure the low PHP versions can be passed during Travis CI build, add the `php-5.3` and `php-5.4` version tests on `.travis.yml` setting.